python/dgl/model_zoo/chem/dgmg.py

```diff
@@ -14,10 +14,10 @@
 import dgl
 from dgl import DGLGraph
 
-try:
-    from rdkit import Chem
-except ImportError:
-    pass
+# try:
```

Comment:
Why do you comment out the `try except` clause?

Reply:
Generally speaking, we should avoid using this to limit the import scope in the future. I'll revert back this and seek better solution later.